Makefile

```diff
@@ -56,6 +56,9 @@ submodule:
 install: submodule ## install into Python envirnoment
 	python setup.py install && cd cwl-runner && python setup.py install
 
+dependencies-update:
+	poetry update && poetry export --without-hashes  -f requirements.txt  --output requirements.txt && poetry export --dev --without-hashes  -f requirements.txt  --output dev-requirements.txt
```

Comment:
Do these commands need to be chained with `&&` or can they be on separate lines?